### PR TITLE
[stable10] drone adding oracle phpunit

### PR DESCRIPTION
**Pull request by SergioBertolinSG**
Backport of https://github.com/owncloud/core/pull/29534

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@SergioBertolinSG we should finish the clean up of scripts before backporting

**Comment by SergioBertolinSG** [comment]
This PR in master was previous to the clean up one. I think the right choice is to backport this one now and later backport the clean up one in the same order as they were/will be merged in master.

File: Jenkinsfile [hunk 1]
```diff
@@ -19,27 +19,6 @@ timestampedNode('SLAVE') {
...
-        executeAndReport('tests/autotest-results-oci.xml', currentStage) {
-            sh '''
-            export NOCOVERAGE=1
-            unset USEDOCKER
-            phpenv local 5.6
-            make test-php TEST_DATABASE=oci
-            '''
-        }
-
-    stage 'Files External: webdav'
```
Comment (DeepDiver1975) [inline Jenkinsfile]: no :-)

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

**Comment by PVince81** [comment]
what's missing ? failed rebase ?

**Comment by patrickjahns** [comment]
closing in favor of #30728 - which contains oracle test runs

**Pull request closed without merging**